plaso/parsers/dsv_parser.py

```diff
@@ -147,13 +144,19 @@ def ParseFileObject(self, parser_mediator, file_object, **unused_kwargs):
           '[{0:s}] Unable to parse DSV file: {1:s} size of file exceeds '
           'maximum supported size').format(self.NAME, display_name))
 
-    line_reader = line_reader_file.BinaryLineReader(file_object)
+    if not self._encoding:
+      self._encoding = parser_mediator.codepage
```

Comment:
I opt not to set `self._encoding` here, seeing it changes the state of the object, maybe define encoding as a local variable instead?

```
encoding = self._encoding or parser_mediator.codepage
```

Reply:
I agree with what you're getting at here, but this is currently needed to support the way some parsers work, notably McAfee. Filed #1971 for this.